### PR TITLE
Components Rendering: Support nested readmes

### PR DESCRIPTION
**Pull request by Benbentwo**
## what
* Update Component Renderer to support other Markdown files, such as `docs/some-doc.md`
  * ignores Changelog, since that is added to the README.md specifically (at the bottom)
  * supports copying images to keep local references

## why
* Components can have a docs folder, this is a common pattern. This is a pattern we will be implementing in [terraform-aws-components](https://github.com/cloudposse/terraform-aws-components) 
* This pattern will continue as we move components to the `cloudposse-components` GitHub organization.

### CODE REVIEW

File: scripts/docs-collator/ComponentRenderer.py [hunk 1]
```diff
@@ -43,11 +51,12 @@ def __render_doc(self, component, file):
...
 
-        label = component
-        title = component
+        name = component if os.path.basename(file) == "README.md" else os.path.basename(file).replace(".md", "")
+        label = name
+        title = name
         github_edit_url = f"https://github.com/{GITHUB_REPO}/blob/master/modules/{relative_path}"
 
-        if len(relative_path.split('/')) > 2:  # this is submodule
+        if len(relative_path.split('/')) > 2 and relative_path.split('/')[1] != 'docs' :  # this is submodule
```
Comment (Benbentwo) [inline scripts/docs-collator/ComponentRenderer.py:59]: This smells. Though I'm not sure how we want to handle this, we need to enforce either `docs/` or `modules` but think of tgw which uses `tgw/hub` as a component, it is nested, therefore I stuck with `docs` as an exception to the rule (where it's not a directory of TF)

File: README.yaml [hunk 2]
```diff
@@ -43,3 +43,5 @@ contributors:
     github: "goruha"
   - name: "Andriy Knysh"
     github: "aknysh"
+  - name: "Benjamin Smith"
+    github: "Benbentwo"
```
Comment (milldr) [inline README.yaml:47]: FYI this doesnt do anything. Contributors are pulled from `contribs`. But doesnt hurt either

Comment (milldr) [reply README.yaml:47]: (with the updated README template)

Comment (Benbentwo) [reply README.yaml:47]: Yeah added mostly to trigger a build

**Review by milldr — APPROVED** [review]

**Pull request merged**